packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -168,3 +181,65 @@ test('ICA connection can send msg with proto3', async t => {
     message: 'ABCI code: 5: error handling packet: see events for details',
   });
 });
+
+test('Query connection can be created', async t => {
+  const {
+    runUtils: { EV },
+  } = t.context;
+
+  const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
```

Comment:
this is `any`. please annotate
```suggestion
  const orchestration: Orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
```

We could encode that upstream but `EV` is at a lower level than the vat set so we'd have to have a custom EV type for this package